app/src/components/AddFeedbackForm/index.js

```diff
@@ -0,0 +1,166 @@
+import React, { Component, PropTypes } from 'react';
+import Form from 'grommet-udacity/components/Form';
```

Comment:
I change the imports so that the component name is uppercase.  [See here for details](https://github.com/grommet/grommet/blob/master/src/js/index.js).  The component imports must match exactly, or the app won't compile on unix systems.